frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card-item.tsx

```diff
@@ -53,11 +60,13 @@ export const SavingsItem: React.FC<SavingsItemProps> = React.memo(
     const infoText =
       'Savings shows the uncompressed and non-deduped data that would have been stored without those techniques';
     let stats: string = null;
-    if (!_.isNil(savings)) {
+    const savingsValue = Number(savings);
+    if (savings) {
+      const savedBytes = humanizeBinaryBytesWithoutB(savingsValue).string;
       const savingsPercentage = logicalSize
-        ? `(${humanizePercentage((100 * Number(savings)) / logicalSize).string})`
-        : '';
-      stats = `${humanizeBinaryBytesWithoutB(savings).string} ${savingsPercentage}`;
+        ? `${savedBytes} (${humanizePercentage((100 * savingsValue) / Number(logicalSize)).string})`
+        : logicalSize;
+      stats = savingsValue <= 0 ? 'No Savings' : savingsPercentage;
```

Comment:
```suggestion
if (savings && logicalSize) {
      const savingsValue = Number(savings);
      const savedBytes = humanizeBinaryBytesWithoutB(savingsValue).string;
      const savingsPercentage = `${savedBytes} (${
        humanizePercentage((100 * savingsValue) / Number(logicalSize)).string
      })`;
      stats = savingsValue <= 0 ? 'No Savings' : savingsPercentage;
    ```

Reply:
Can you add a comment there, `humanizePercentage` checks for definiteness, for future references and maintenance.

Reply:
I think these are exactly the same. The only difference is the style, right?

Reply:
We need to do calculations when we have both `savings` and `logicalSize` available